### DISCUSSION TITLE
Jupyter Notebook alternative to maintain the state of simulation

### DISCUSSION THREAD

**Discussion by TheASingh** [CURENT/andes#347] [Q&A]
Dr. Cui,

I'm done with my RnD on ANDES simulation using Juptyer notebook. Now, I'm looking for an alternative of Jupyter to run the simulation and also able to resume the simulation where it ended previously (in other words, I want to maintain the state of simulation but not on Jupyter).

**Comment by cuihantao** [comment]
I really have no idea. Why don’t you just ask Google as the question is general to any package? Please respect my time and not to use the discussion board as the go-to place for questions in which yourself have to invest time.

Regards,
Hantao Cui
